packages/components/src/Icon/Icon.scss

```diff
@@ -15,6 +15,10 @@ $tc-icon-size: $svg-md-size !default;
 	}
 }
 
+.link {
```

Comment:
what is the point of this modification since the icon can be already wrapper in a link where it is used ? 
https://github.com/Talend/ui/blob/master/packages/components/src/Icon/Icon.component.js#L75

Reply:
a tag with xlinkHref attribute does not provide the right pointer, so I enforced it.